safe/reportv4/impact_report.py

```diff
@@ -252,7 +252,7 @@ def __init__(
             analysis=None,
             exposure_breakdown=None,
             aggregation_impacted=None,
-            extra_layers=[],
+            extra_layers=None,
```

Comment:
Well, previously I didn't check it. 
But it seems we need to use immutable default argument after some google and read. One of the article: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments